envi/archs/thumb16/disasm.py

```diff
@@ -970,6 +977,8 @@ class Thumb16Disasm:
     _opclass = ThumbOpcode
 
     def disasm(self, bytez, offset, va, trackMode=True):
+        flags = 0
+        va &= -2
```

Comment:
Architecturally, all opcodes in arm/thumb *MUST* be 16-bit aligned.  arm is 32-bit aligned.  not tracking flags was f*ing up tracking opcode flags, which lead to problems with code-flow traversal.